### ISSUE TITLE
Rename "dashboard" of the datastore to "status" or similar

### ISSUE THREAD

**Issue by michaelwood** [ThreeSixtyGiving/datastore#194]
Aim to reduce the congestion on the word "dashboard" as we have the Quality dashboard API as a module within the datastore.

**Comment by BibianaC** [comment]
@michaelwood @R2ZER0 I have renamed 'dashboard' to 'status' in the HTML files (#266 ). Was this what you were expecting or did you want to rename it also in the back end?

**Comment by michaelwood** [comment]
@BibianaC yes please do go ahead and update it in the backend, (in the relevant `urls.py`) . [views](https://github.com/ThreeSixtyGiving/datastore/blob/live/datastore/ui/views.py#L87C7-L87C20), `DashBoardView` and the template file name too. (`git mv`) to preserve the git history.

**Comment by BibianaC** [comment]
Done.
